images/installer/root/usr/local/bin/entrypoint-provider

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
You need to leave the existing entrypoint in place until we migrate the older jobs.  Create a shim that sets TYPE and comment to that effect.

You'll also need to backport this change to release-3.9 for the GCP jobs to keep working.